assets/js/modules/sign-in-with-google/utils/provisioning.js

```diff
@@ -77,5 +78,23 @@ export function sanitizeProvisioningParams( params ) {
 
 	sanitizedParams.appname = sanitizedAppname;
 
+	// Replace all non-alphanumeric characters with spaces in sitename.
```

Comment:
Regarding above part that I can't comment on directly:

```js
md5( host ).substring( 0, 16 ); // Truncate MD5 to fit in 30 chars.
```
The substring here isn't necessary (it's also a duplicate statement) since we trim the whole app name afterwards if it's too long, so may as well just do that once.